code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -309,21 +312,36 @@
 
 /datum/reagent/mercury
 	name = "Mercury"
-	description = "A chemical element."
+	description = "A poisonous chemical element, one of two that is a liquid at human room temperature and pressure."
 	reagent_state = LIQUID
 	color = "#484848"
-	ingest_met = REM*0.2
+	ingest_met = REM*0.1
+	breathe_met = REM*0.4 
+	breathe_mul = 2 //mercury vapours and skin absorption more dangerous than eating mercury.
+	touch_met = REM*0.1
+	touch_mul = 1.25
 	taste_mult = 0 //mercury apparently is tasteless
+	scannable = 1
```

Comment:
These instances of scannable = 1 should be scannable = TRUE

Reply:
All done.